package.json

```diff
@@ -15,16 +15,18 @@
 		".": {
 			"node": {
 				"types": "./index.d.ts",
-				"import": "./index.js"
+				"import": "./index.js",
+				"module-sync": "./index.js"
 			},
 			"default": {
 				"types": "./core.d.ts",
-				"import": "./core.js"
+				"import": "./core.js",
+				"module-sync": "./core.js"
 			}
 		},
-		"./core": {
-			"types": "./core.d.ts",
-			"import": "./core.js"
+		"./node": {
+			"types": "./index.d.ts",
+			"default": "./index.js"
 		}
```

Comment:
Not worth doing a breaking change for this. So just add `node` and keep `core`, and we can remove `core` in the next major version.